formats/organisation/publisher/links.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This file isn't required. If it's not provided, the schema will be generated with an empty links hash.